docs/guides/functions/configuring-lambda.md

```diff
@@ -0,0 +1,150 @@
+---
+title: Configuring Lambda function settings
+description: How to configure custom settings for your Lambda function
+---
+
+In this guide you will learn how to configure custom settings for a Lambda function.
+
+### Overview
+
+When creating a function within an Amplify project, you are given several options to choose how you would like your function to be configured. You may find that your application requires a different configuration than the CLI set up for you.
```

Comment:
```suggestion
You may want to override the Amplify CLI default configurations for your Lambda function or configure changes not available within the `amplify add function` workflow.
```
Maybe something like this to keep it shorter. Then you can collapse all of this with the primary sentence on top.